### PR TITLE
Set line endings for bash scripts

### PR DESCRIPTION
**Pull request by nickchak21**
Set line endings in the gitattributes for bash scripts. Also cleared up where build.sh is located in Docker container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ The following needs to be done once at the beginning, and then later only when m
 #### Executing Build Script
 We use a custom build script that simplifies the process of building and overlaying catkin workspaces.
 
-To build our workspaces for ROS, in the `robosub-ros` directory, execute
+To build our workspaces for ROS, in the `/dev/robosub-ros` directory, execute
```
Comment (muthuArivoli) [inline README.md:160]: This is technically incorrect. It should be either `~/dev/robosub-ros` or `/root/dev/robosub-ros`

**Review by muthuArivoli — CHANGES_REQUESTED** [review]
See change

File: README.md (hunk 1 above)
Comment (nickchak21) [reply README.md]: I made the change

**Comment by rabinowitz** [comment]
I like the folder path clarification.

I don't see the autocrlf change in the Files Changed tab on this PR, but it was made in one of the commits. Is it possible you undid it by mistake?

Further, I think we should be able solve the problem without changing the readme, but instead by adding to the `.gitattributes` file, as you can see I did [here](https://github.com/DukeRobotics/robosub-ros/blob/task-planning/.gitattributes#L1) in the task-planning branch.

**Comment by muthuArivoli** [comment]
We removed autocrlf, since we don't want to change a user's system git settings just for our project. Agreed that the gitattributes is the correct solution, will add that to this PR.

**Review by rabinowitz — APPROVED** [review]
LGTM

**Review by muthuArivoli — APPROVED** [review]
LGTM, merging now. Future README updates should only be package specific, so bundle them up in an existing or upcoming PR that is targeting that package only.

**Pull request merged**